js/annotationtoolbar.mjs

```diff
@@ -14,18 +14,24 @@ import {TransformTool} from './papertools/transform.mjs';
 import {RasterTool} from './papertools/raster.mjs';
 
 export class AnnotationToolbar{
```

Comment:
It would be great to document what the class itself actually does with a class description. See the [OpenSeadragon](https://github.com/openseadragon/openseadragon) source code for an example (like here: https://github.com/openseadragon/openseadragon/blob/master/src/navigator.js). Note that those "classes" were implemented before classes were a real javascript construct so it might work a little differently - but probably not much.